### PR TITLE
Deprecate sendToDevice and sendToDeviceGroup and their response classes

### PR DESCRIPTION
**Pull request by Doris-Ge**
* Deprecate `sendToDevice` and its response class, `MessagingDeviceResponse`
* Deprecate `sendToDeviceGroup` and its response class, `MessagingDeviceGroupResponse`

### CODE REVIEW

**Review by lahirumaramba — APPROVED** [review]
Thank you @Doris-Ge ! LGTM!
Let's get a TW review as well.

File: src/messaging/messaging-api.ts [hunk 1]
```diff
@@ -940,7 +940,11 @@ export interface MessagingOptions {
   [key: string]: any | undefined;
 }
 
-/* Individual status response payload from single devices */
+/**
+ * Individual status response payload from single devices
+ *
+ * @deprecated Returned by {@link Messaging#sendToDevice} which is also deprecated.
```
Comment (egilmorez) [inline src/messaging/messaging-api.ts:946]: Nit, but "which" is generally preceded by a comma.  Suggest one here and for similar/identical instances below.

**Review by egilmorez — APPROVED** [review]
LG with a nit, thanks!

File: src/messaging/messaging-api.ts (hunk 1 above)
Comment (Doris-Ge) [reply src/messaging/messaging-api.ts]: Good catch! Thanks! Done.

**Pull request merged**